### DISCUSSION TITLE
Worklflow_dispatch without inputs

### DISCUSSION THREAD

**Discussion by mv185095** [community/community#26098] [Actions]
<p>How to make a <code>workflow_dispatch</code> without any inputs?</p>
<pre><code class="lang-auto">on: 
  workflow_dispatch: []
</code></pre>
<p>results in infinite “loading” of the workflow dispatch dialog. Without any parenthesis it is invalid syntax. I feel like I am missing something obvious.</p>
<p><img src="https://user-images.githubusercontent.com/67328110/181095220-00f67e72-ea5c-4b6d-b045-236060f7d3d9.png" alt="image" data-base62-sha1="bi5BZeLzo7vwjtRrOZX9w9zTCeo" width="325" height="178"></p>

**Comment by weide-zhou** [comment, marked as answer]
<p>Hi <a class="mention" href="/mv185095">@mv185095</a>,</p>
<p>Pleases remove <code>[]</code> , just use below code, it works without any inputs.</p>
<pre><code class="lang-auto">on:
  workflow_dispatch: 
</code></pre>
<p>Thanks</p>

**Reply by berkon** [reply]
But eslint gives the following error in this case, so it doesn't seem to be correct:

**error Empty mapping values are forbidden  yml/no-empty-mapping-value**

**Reply by LeaveTheCapital** [reply]
Hi @berkon It is valid workflow syntax as mentioned in the official docs here:
   https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#example-of-jobsjob_idsecretsinherit

While it may often be against best practice to include an empty block like this, it is the recommended approach in this scenario.
If you want to preserve this linting rule, you can ignore specific instances like this:

on:
  # eslint-disable-next-line yml/no-empty-mapping-value
  workflow_call:

**Reply by berkon** [reply]
I found out that using {} instead of [] works fine!
